### ISSUE TITLE
structs.unpack doesn't work properly

### ISSUE THREAD

**Issue by humhue** [shedskin/shedskin#367]
First, I'd like to express my appreciation for this project, it's great this is still being worked on.
Anyway, I'm having some troubles compiling a certain source with Shed Skin.

MRE:
```
import struct

def pack_u32(n):
    return struct.pack("<I", n) # the packing works fine

def unpack_u32(n):
    res, _ = struct.unpack("<I", n) # this doesn't
    return res

print(unpack_u32(pack_u32(12000)))
# this should output 12000, but outputs -32
```

I have no idea why this is the case. I'm using the latest master.

**Comment by srepmub** [comment]
thank you for your kind words, and even more for reporting this issue!

not sure when this got broken (it should have worked at some point.... I hope), but I just commited a fix. we would love to hear it if you run into any other issue..

is the 'certain source' something interesting you could share? =)

PS the given example doesn't work in python either:  'res, _' should probable be 'res,' here.

**Comment by humhue** [comment]
Thanks! That was fast!
This works as intended now. But I'm still experiencing issues, this time with fairly new syntax: the walrus operator added in 3.8. I'll open a separate issue for that one.
Regarding the source, it's nothing crazy, just an optimized version of the code I used to find the TLKStrings of files in Bioware Generic File Format, in my only repo.
Ideally, I'd rewrite that code and build a GUI out of it, and I figured it would be really cool to do that in Python instead of using another language entirely, like, say, Nim. This is why I like Shed Skin.
Anyway, here's the code I'm trying to compile if you're curious: https://gist.github.com/humhue/05c38f0c1b40f3043e6bec76a31a6d19
